packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.html

```diff
@@ -0,0 +1,103 @@
+<h1 data-translate="datacenter_backup_title"></h1>
+<div class="datacenter-backup" data-ui-view>
+    <div data-ovh-alert="{{ $ctrl.alertMain }}" id="backupMainAlert"></div>
+    <oui-switch data-model="true" data-on-change="$ctrl.deactivate()">
+    </oui-switch>
+    <span data-translate="datacenter_backup_enabled"></span>
+
+    <section class="datacenter-backup-offers">
+        <h4 class="mb-3" data-translate="datacenter_backup_choose_offers"></h4>
+        <dedicated-cloud-datacenter-backup-offers
+            data-backup-offers="$ctrl.backupOffers"
+            data-backup="$ctrl.backup"
+            data-preview-mode="false"
+            data-user="$ctrl.currentUser"
+        ></dedicated-cloud-datacenter-backup-offers>
+        <oui-button
+            data-variant="primary"
+            data-text="{{ 'datacenter_backup_update_offer' | translate }}"
```

Comment:
I think `text` attribute is deprecated, we should use a nested `span` instead

Reply:
This is moved to a span